editor/plugins/canvas_item_editor_plugin.h

```diff
@@ -216,7 +216,8 @@ class CanvasItemEditor : public VBoxContainer {
 	// Defaults are defined in clear().
 	Point2 grid_offset;
 	Point2 grid_step;
-	int primary_grid_steps = 0;
+	int primary_grid_step_x = 0;
+	int primary_grid_step_y = 0;
```

Comment:
Shouldn't this be Vector2i? 🤔
Doesn't matter much, but other grid steps are vector.

Reply:
Yes, will change.